pkg/cloudprovider/providers/azure/azure_test.go

```diff
@@ -299,7 +299,7 @@ func TestReconcileSecurityGroupNewServiceAddsPort(t *testing.T) {
 
 	sg := getTestSecurityGroup()
 
-	sg, _, err := az.reconcileSecurityGroup(sg, testClusterName, &svc1, true)
+	sg, _, err := az.reconcileSecurityGroup(sg, testClusterName, &svc1, to.StringPtr("13.70.140.150"), true)
```

Comment:
Is this an acceptable use of a hard-coded IP?

Reply:
It's in unit tests, so it is never actually used as an IP address.  But I will change it to something meaningless / invalid - the tests will still work.

Reply:
I would use an rfc 1918 address.